tsconfig.sdk.json

```diff
@@ -1,6 +1,10 @@
 {
   "extends": "./tsconfig.base.json",
   "compilerOptions": {
+    // should make tsc run faster, we may need to delete the buildinfo file if we run into issues
+    "incremental": true,
```

Comment:
This should make tsc run *a lot* faster after the first run.
It saves some metadata on a .tsbuildinfo file to know which files it already "build".

Reply:
Awesome! Should we add `.tsbuildinfo` to the `.gitignore` as well?

Reply:
The whole `/resources/embedding-sdk` folder is gitignored already